main.go

```diff
@@ -31,6 +32,7 @@ type Config struct {
 
 	IpaPath           string `env:"ipa_path"`
 	PkgPath           string `env:"pkg_path"`
+	Platform          string `env:"platform,opt[ios,macos,tvos]"`
```

Comment:
Could we detect this instead?

Reply:
Idea: [**auto**,ios,macos,tvos]. With auto the step should try to figure out the platform.